src/gfx/reverse.cpp

```diff
@@ -199,6 +222,22 @@ void reverse() {
 
 		if (options.palSpecType == Options::EXPLICIT && palettes != options.palSpec) {
 			warning("Colors in the palette file do not match those specified with `-c`!");
+			fputs("Colors specified in the palette file:         ...and with `-c`:\n", stderr);
+			for (decltype(palettes)::size_type i = 0;
```

Comment:
```suggestion
			for (size_t i = 0;
```

Reply:
Not strictly guaranteed, but eh, I think we can take that.